tests/integration_tests/functional/test_pause_resume.py

```diff
@@ -1,65 +1,108 @@
+    assert microvm.api_session.is_status_bad_request(response.status_code)
 
-    # Start microVM.
-    test_microvm.start()
+    microvm.start()
 
-    ssh_connection = net_tools.SSHConnection(test_microvm.ssh_config)
+    ssh_connection = net_tools.SSHConnection(microvm.ssh_config)
 
     # Verify guest is active.
     exit_code, _, _ = ssh_connection.execute_command("ls")
     assert exit_code == 0
 
     # Pausing the microVM after it's been started is successful.
-    response = test_microvm.vm.patch(state='Paused')
-    assert test_microvm.api_session.is_status_no_content(response.status_code)
+    response = microvm.vm.patch(state='Paused')
+    assert microvm.api_session.is_status_no_content(response.status_code)
 
     # Verify guest is no longer active.
     exit_code, _, _ = ssh_connection.execute_command("ls")
     assert exit_code != 0
 
     # Pausing the microVM when it is already `Paused` is allowed
     # (microVM remains in `Paused` state).
-    response = test_microvm.vm.patch(state='Paused')
-    assert test_microvm.api_session.is_status_no_content(response.status_code)
+    response = microvm.vm.patch(state='Paused')
+    assert microvm.api_session.is_status_no_content(response.status_code)
 
     # Resuming the microVM is successful.
-    response = test_microvm.vm.patch(state='Resumed')
-    assert test_microvm.api_session.is_status_no_content(response.status_code)
+    response = microvm.vm.patch(state='Resumed')
+    assert microvm.api_session.is_status_no_content(response.status_code)
 
     # Verify guest is active again.
     exit_code, _, _ = ssh_connection.execute_command("ls")
     assert exit_code == 0
 
     # Resuming the microVM when it is already `Resumed` is allowed
     # (microVM remains in the running state).
-    response = test_microvm.vm.patch(state='Resumed')
-    assert test_microvm.api_session.is_status_no_content(response.status_code)
+    response = microvm.vm.patch(state='Resumed')
+    assert microvm.api_session.is_status_no_content(response.status_code)
 
     # Verify guest is still active.
     exit_code, _, _ = ssh_connection.execute_command("ls")
     assert exit_code == 0
+
+    microvm.kill()
+    del tap
+
+
+@pytest.mark.skipif(
+    platform.machine() != "x86_64",
+    reason="Not supported yet."
+)
+def test_pause_resume(test_session_root_path,
+                      network_config,
+                      bin_cloner_path):
+    """Test scenario: boot/pause/resume for all available configurations."""
+    logger = logging.getLogger()
+    # Currently, artifacts share the bucket with all other resources.
+    artifacts = ArtifactCollection(test_images_s3_bucket())
+
+    microvm_artifacts = artifacts.microvms()
+    kernel_artifacts = artifacts.kernels()
+    disk_artifacts = artifacts.disks()
+
+    # Create a test matrix. Push logger and network as context variables.
+    test_matrix = TestMatrix(context={
+            'network_config': network_config,
+            'logger': logger
```

Comment:
I think that you could give a name to the logger here. What do you think?
You can _name_ logger instances by passing a string to the `getLogger` call, in order to have all log entries marked with the given name.

Reply:
done.